projects/igniteui-angular/src/lib/grids/grid-base.component.ts

```diff
@@ -2749,7 +2749,11 @@ export abstract class IgxGridBaseComponent extends DisplayDensityBase implements
         });
 
         this.resizeNotify.pipe(destructor, filter(() => !this._init), throttleTime(40))
-            .subscribe(() => this.notifyChanges(true));
+            .subscribe(() => {
+                this.zone.run(() => {
```

Comment:
Not a fan of triggering a change detection cycle with a chatty observer

Reply:
In some cases, the resize observer is fired, but no change detection is triggered until you do some action on the grid (click a cell for example).  In those cases the grid appears broken (shows white spaces) until the next change detection cycle which updates the sizes. You can view such a scenario for example in the sample attached in issue #4952.

Reply:
Then I suggest we increase the throttle time as having a spammable observer inside zone run is taxing on the UI thread